File: _Contabilidad/MantenimientoCtasContables/CtbValida.cs
using System;
using Microsoft.VisualBasic;

namespace MantCtb
{
    public partial class CtbValida
    {
        public CtbValida()
        {
            InitializeComponent();
            _BtnImprimir.Name = "BtnImprimir";
            _btnSalir.Name = "btnSalir";
        }

        private void imprimir_Click(object sender, EventArgs e)
        {
            var imp = new DataGridViewPrinterApplication1.frmMain();
            imp.imprimir(Malla, "Errores en configuración del plan de cuentas contables " + Strings.Format(DateTime.Now, "dd/MMM/yyyy"), Empresa: SysEmpDatt.datosEmpresa.Emp_Nombre);
            imp.Dispose();
        }

        private void btnSalir_Click(object sender, EventArgs e)
        {
            Close();
        }
    }
}

File: _Contabilidad/MantenimientoCtasContables/CtbValida.vb
Public Class CtbValida
    Private Sub imprimir_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles BtnImprimir.Click
        Dim imp As New DataGridViewPrinterApplication1.frmMain
        imp.imprimir(Malla, "Errores en configuración del plan de cuentas contables " + Format(Date.Now, "dd/MMM/yyyy"), , DattCom.datosEmpresa.Emp_Nombre)
        imp.Dispose()
    End Sub

    Private Sub btnSalir_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles btnSalir.Click
        Me.Close()
    End Sub
End Class